state/cluster/badger/mutator_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	pbadger "github.com/onflow/flow-go/state/protocol/badger"
 	"github.com/onflow/flow-go/state/protocol/events"
 	"github.com/onflow/flow-go/state/protocol/inmem"
+	util2 "github.com/onflow/flow-go/state/protocol/util"
```

Comment:
if this needs a package alias, maybe we can give it a more descriptive one like `protocolutil`